src/library/scala/math/package.scala

```diff
@@ -215,9 +218,37 @@ package object math {
     */
   def signum(x: Long): Long     = java.lang.Long.signum(x)
   /** @group signum */
-  def signum(x: Float): Float   = java.lang.Math.signum(x)
+  def signum(f: Float): Float   = java.lang.Math.signum(f)
   /** @group signum */
-  def signum(x: Double): Double = java.lang.Math.signum(x)
+  def signum(d: Double): Double = java.lang.Math.signum(d)
+
+  def floorDiv(x: Int, y: Int): Int = java.lang.Math.floorDiv(x, y)
+
+  def floorDiv(x: Long, y: Long): Long = java.lang.Math.floorDiv(x, y)
+
+  def floorMod(x: Int, y: Int): Int = java.lang.Math.floorMod(x, y)
+
+  def floorMod(x: Long, y: Long): Long = java.lang.Math.floorMod(x, y)
+
+  def copySign(magnitude: Double, sign: Double): Double = java.lang.Math.copySign(magnitude, sign)
+
+  def copySign(magnitude: Float, sign: Float): Float = java.lang.Math.copySign(magnitude, sign)
+
+  def nextAfter(start: Double, direction: Double): Double = java.lang.Math.nextAfter(start, direction)
+
+  def nextAfter(start: Float, direction: Double): Float = java.lang.Math.nextAfter(start, direction)
+
+  def nextUp(d: Double): Double = java.lang.Math.nextUp(d)
+
+  def nextUp(f: Float): Float = java.lang.Math.nextUp(f)
+
+  def nextDown(d: Double): Double = java.lang.Math.nextDown(d)
+
+  def nextDown(f: Float): Float = java.lang.Math.nextDown(f)
+
+  def scalb(d: Double, scaleFactor: Int): Double = java.lang.Math.scalb(d, scaleFactor)
```

Comment:
@golem131 - Adding a new group called _Exact Scaling_ only for `scalb` is too rich for my pipe so I'm suggesting you group this pair into the `exact` group.